src/main/programming/logic_condition.h

```diff
@@ -82,8 +82,9 @@ typedef enum {
     LOGIC_CONDITION_DELTA                       = 50,
     LOGIC_CONDITION_APPROX_EQUAL                = 51,
     LOGIC_CONDITION_LED_PIN_PWM                 = 52,
-    LOGIC_CONDITION_DISABLE_GPS_FIX             = 53,
-    LOGIC_CONDITION_LAST                        = 54,
+    LOGIC_CONDITION_RESET_MAG_CALIBRATION       = 53,
+    LOGIC_CONDITION_DISABLE_GPS_FIX             = 54,
```

Comment:
Your entry has to have a new ID, not reuse existing one. So, `LOGIC_CONDITION_RESET_MAG_CALIBRATION` should be `54` not `53`